src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/RunListenerImpl.java

```diff
@@ -94,7 +94,7 @@ public void onStarted(Run r, TaskListener listener) {
             json.put(Util.KEY_CAUSES, causes.toString());
             List<String> parameters = new LinkedList<String>();
             for (String buildKey : build.getBuildVariables().keySet()) {
-                parameters.add(buildKey + "=" + build.getBuildVariables().get(buildKey));
+                parameters.add(0, buildKey + "=" + build.getBuildVariables().get(buildKey));
```

Comment:
We seem to get the parameters in a non-defined order, so reversing the list doesn't help us.